src/sentry/integrations/opsgenie/utils.py

```diff
@@ -84,6 +96,8 @@ def send_incident_alert_notification(
     )
     client = install.get_keyring_client(keyid=team["id"])
     attachment = build_incident_attachment(incident, new_status, metric_value, notification_uuid)
+    attachment = attach_custom_priority(attachment, action)
```

Comment:
Is there a reason you're not calling `attach_custom_priority` within `build_incident_attachment`?

Reply:
i guess i could. however, we don't currently pass the `action` into `build_incident_attachment` and this same function is declared in multiple places and i didn't want to change the convention